docs/idea.rst

```diff
@@ -44,14 +43,14 @@ For RestrictedPython this ``compile()`` method is replaced by:
 
 .. code-block:: python
 
-    compile_restricted(source, filename, mode [, flags [, dont_inherit]])
+    RestrictedPython.compile_restricted(source, filename, mode [, flags [, dont_inherit]])
 
-The primary parameter ``source`` has to be a ASCII or ``unicode`` string (With Python 2.6 an additional option for source was added: ``ast.AST`` for :ref:`Code generation <_sec_code_generation>`).
-Both methods either returns compiled byte code that the interpreter could execute or raise exceptions if the provided source code is invalid.
+The primary parameter ``source`` has to be a string or ``ast.AST`` instance.
+Both methods either return compiled byte code that the interpreter can execute or raise exceptions if the provided source code is invalid.
 
 As ``compile`` and ``compile_restricted`` just compile the provided source code to byte code it is not sufficient to sandbox the environment, as all calls to libraries are still available.
```

Comment:
Maybe I do not get the scope of a sandbox here, but this wording seems more reasonable to me. 
```suggestion
As ``compile`` and ``compile_restricted`` just compile the provided source code to byte code it is not sufficient as a sandbox environment, as all calls to libraries are still available.
```